### PR TITLE
Add review page accordions

### PR DESCRIPTION
**Pull request by jasalisbury**
**Description of change**

The review page now has an accordion with an item for each page that matches the desired design. The report still needs to be marked as `Submitted` in the backend after saving of a report (https://github.com/HHS/Head-Start-TTADP/issues/98) is closed.

 * Each page defines sections that are used in the review accordion
 * Simplified how the activity report imports the pages for the report
 * Date picker now accepts a string MM/DD/YYYY and spits out the same
 * Activity report page file names are no longer capitalized because they are no longer react components

**How to test**

Dev:

Test on https://tta-smarthub-dev.app.cloud.gov/activity-reports/review. Note I haven't tested on DEV yet because I can't login to the UAT environment. I didn't want to wait for UAT to work before getting eyes on this PR.

Local:

1. Pull down
2. Run migrations and seeders
3. Go to Activity report and fill out a report
4. Click the accordion in the review page, verify form values are correctly shown

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/135
* closes https://github.com/HHS/Head-Start-TTADP/issues/245

**Checklist**

- [ ] Meets issue criteria
- [x] Code tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

File: src/routes/activityReports/handlers.js
Comment (rahearn) [inline src/routes/activityReports/handlers.js:67]: Just putting a pin in that we will need to add authorization to this route when it is implemented.

File: frontend/src/pages/ActivityReport/Pages/ReviewSubmit.js
Comment (rahearn) [inline frontend/src/pages/ActivityReport/Pages/ReviewSubmit.js:86]: I can't figure out why - but when I run this locally it has a weird default value on the approvers list.

Screen Shot 2021-01-05 at 5 42 49 PM

File: frontend/src/pages/ActivityReport/Pages/topicsResources.js
Comment (rahearn) [inline frontend/src/pages/ActivityReport/Pages/topicsResources.js:101]: The URL resources don't appear in the review page.

**Review by rahearn — CHANGES_REQUESTED** [review]
Really love the ability to jump back to the relevant form question from the review page.

In addition to the couple of inline things, please add some api documentation for the approvers api endpoint.

File: frontend/src/pages/ActivityReport/Pages/ReviewSubmit.js
Comment (jasalisbury) [reply frontend/src/pages/ActivityReport/Pages/ReviewSubmit.js:86]: I left in some old data simulating loading managers/additional notes from an already saved report. I'll remove the old data.

**Review by rahearn — APPROVED** [review]
Leaving #135 open until actually submitting for approval is complete, but the review page is 👍🏻

**Pull request merged**
